app/controllers/agents_controller.rb

```diff
@@ -18,15 +18,17 @@ def show
   end
 
   def ajax_agents
-    filters = { name: params[:name] }
```

Comment:
why remove this?

Reply:
same where is `filters[:agentType]` ?

Reply:
See the remarks about the query parmeter in https://github.com/ontoportal-lirmm/ontologies_api/pull/86

Reply:
filter[:agentType] has no sense cause we don't need to search by agent type anymore (we remove those constraints in the back)